### PR TITLE
Add handler and fix typo

### PR DESCRIPTION
**Pull request by Petrprogs**
1. Add handler for cache write fail
2. Fix typo in word "response"

### CODE REVIEW

File: src/mahomaps/overlays/CacheFailedOverlay.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+public class CacheFailedOverlay extends MapOverlay implements IButtonHandler {
+
+	public CacheFailedOverlay() {
+		content = new FillFlowContainer(new UIElement[] { new SimpleText("Caching failed!"),
+				new SimpleText("Please, check that memory is not full."), new ColumnsContainer(new UIElement[] {
+						new Button(MahoMapsApp.text[38], 0, this) }) });
+	}
+
+	public String GetId() {
+		return "cache_fail";
```
Comment (Feodor0090) [inline src/mahomaps/overlays/CacheFailedOverlay.java:23]: Overlays with same id will discard each other, this behaviour is used in route-related overlays, for example. Here, we probably don't want to mess it with others.

**Review by Feodor0090 — APPROVED** [review]
Yeah, something like that. Thanks for your help.

Can you provide feedback about our overlays UI system? Isn't it too weird? Are there things to be refactored by your opinion?

**Comment by Feodor0090** [comment]
I fixed workflow config, all future PRs will run build as it was expected. This one will be left without checks <s>because nobody actually cares about it</s>

**Comment by Petrprogs** [comment]
> Yeah, something like that. Thanks for your help.
> 
> Can you provide feedback about our overlays UI system? Isn't it too weird? Are there things to be refactored by your opinion?

The overlay system is nice, but can you make translation files as json? Because finding a specific line in a txt file is not very convenient :).

**Pull request merged**